bowtie/benchmarks/contains.py

```diff
@@ -0,0 +1,21 @@
+def get_benchmark():
```

Comment:
Might want to split off a separate PR to add this `bowtie/benchmarks` directory with all the benchmarks, that might be easier to immediately add? Up to you though.